app/Http/Controllers/Item/ItemController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\Item\ItemRepository;
+use App\Item;
+use app\Http\Requests\CreateItemRequest;
+
+class ItemController extends Controller
+{
+    private $itemRepository;
+
+    public function __construct(ItemRepository $itemRepository)
+    {
+        $this->itemRepository = $itemRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // TO-DO
+        // Ask fro help from Jeseph of how to use the config values on pagination
+        // get the pagination number or a default
+        $items = $this->$itemRepository ?? 10;
+        return view('items' . config('settings.pagination.small') . '.index')->with(['$items' => $items]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('items.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  CreateItemRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CreateItemRequest $request)
+    {
+        $data = $request->all();
+        Item::create($data);
```

Comment:
```suggestion
        Item::create($request->all());
```

The `$data` variable is useless since you are not using it anywhere else.

Reply:
Okay thanks